### ISSUE TITLE
[Bug Report] v-tooltip inside v-card-actions breaks layout of block v-btn

### ISSUE THREAD

**Issue by Toilal** [vuetifyjs/vuetify#3386]
### Versions and Environment
**Vuetify:** 1.0.3
**Vue:** 2.5.13
**Browsers:** Chrome 64.0.3282.167
**OS:** Linux x86_64

### Steps to reproduce
Have a look to the reproduction link

### Expected Behavior
Inside v-card-actions, a v-btn with block attribute inside v-tooltip should grow full width, like the one without tooltip.

### Actual Behavior
Inside v-card-actions, a v-btn with block attribute inside v-tooltip doesn't grow full width, block attribute has no effect.

### Reproduction Link
[https://codepen.io/Toilal/pen/wyYamb](https://codepen.io/Toilal/pen/wyYamb)

**Comment by Toilal** [comment]
This can workaround the issue

```vue
<style scoped lang="scss">
.card__actions >>> .tooltip {
  flex: 1 1 auto;

  > span {
    flex: 1 1 auto
  }
}
</style>
```

**Comment by johnleider** [comment]
Due to how flex box works, and that there is no way for `v-tooltip` to understand that it should be full width, you will have to use a work-around until we remove the need for an element with `v-tooltip`.

https://codepen.io/johnjleider/pen/jZQPbd

**Comment by Toilal** [comment]
Tooltip hell :fire: !

Thanks for quick response and keep the good job, this library is awesome !

![Awesome](https://media.giphy.com/media/IxKt9HOM1mI80/giphy.gif)

**Comment by KaelWD** [comment]
This will eventually be fixed by #3206 if you want something to follow.
